### PR TITLE
Enable sdcard

### PR DESCRIPTION
**Pull request by jlaitine**
Please use [PX4 Discuss](http://discuss.px4.io/) or [Slack](http://slack.px4.io/) to align on pull requests if necessary. You can then open draft pull requests to get early feedback.

**Describe problem solved by this pull request**
A clear and concise description of the problem this proposed change will solve.
E.g. For this use case I ran into...

**Describe your solution**
A clear and concise description of what you have implemented.

**Describe possible alternatives**
A clear and concise description of alternative solutions or features you've considered.

**Test data / coverage**
How was it tested? What cases were covered? Logs uploaded to https://review.px4.io/ and screenshots of the important plot parts.

**Additional context**
Add any other related context or media.

### CODE REVIEW

File: boards/ssrc/saluki/src/CMakeLists.txt [hunk 1]
```diff
@@ -63,6 +64,7 @@ else()
 		toc.c
 		i2c.cpp
 		spi.cpp
+		mpfs_emmcsd.c
```
Comment (jpaali) [inline boards/ssrc/saluki/src/CMakeLists.txt:67]: Does not work for me. Need to add 
${NUTTX_DIR}/boards/risc-v/mpfs/icicle/src/mpfs_emmcsd.c

File: boards/ssrc/saluki/nuttx-config/nsh/defconfig [hunk 2]
```diff
@@ -103,6 +107,9 @@ CONFIG_MAX_TASKS=64
 CONFIG_MCU_SERIAL=y
 CONFIG_MM_DEFAULT_MANAGER=y
 CONFIG_MM_REGIONS=1
+CONFIG_MMCSD=y
+CONFIG_MMCSD_SDIO=y
+CONFIG_MPFS_EMMCSD=y
```
Comment (jpaali) [inline boards/ssrc/saluki/nuttx-config/nsh/defconfig:112]: I need to explicitly set CONFIG_MMCSD_MMCSUPPORT=n
to get Kingston SDCS2/32GB microsd card to work.

Driver tries to identify card with MMC_CMD1 which is not supported and causes timeout.

**Comment by jlaitine** [comment]
Now this is working as expected

File: boards/ssrc/saluki/nuttx-config/nsh/defconfig [hunk 3]
```diff
@@ -257,3 +266,17 @@ CONFIG_SPI=y
 CONFIG_SPI_DRIVER=y
 CONFIG_MPFS_SPI0=y
 
+
```
Comment (jlaitine) [inline boards/ssrc/saluki/nuttx-config/nsh/defconfig:269]: I'll still remove this block

**Comment by jpaali** [comment]
looks fine

**Pull request merged**